webapp/TargetedMS/js/QCPlotHelperBase.js

```diff
@@ -779,7 +750,7 @@ Ext4.define("LABKEY.targetedms.QCPlotHelperBase", {
                     color: '#555555'
                 },
                 yLeft: {
-                    value: this.getYScaleLabel(plotType, trendLineProps.valueConversion, metricProps.series1Label),
+                    value: this.getYScaleLabel(plotType, trendLineProps.valueConversion, metricProps.yAxisLabel1),
```

Comment:
Wouldn't we need to reference yAxisLabel2 somewhere in the JS?

Reply:
yes, added that yRight